### ISSUE TITLE
MLBlocks/Orion support for callbacks/loss plotting

### ISSUE THREAD

**Issue by cjtaylo-csu** [sintel-dev/Orion#284]
* Orion version:   0.2.1dev0
* Python version: 3.6.9
* Operating System: Ubuntu 18.04.6 LTS

### Description

Will MLblocks/Orion add support for callbacks? My main goal is to plot losses during training.
It is a bit hard to do this, as there are multiple layers between the Orion API and the keras .fit() calls.

**Comment by sarahmish** [comment]
Hi @cjtaylo-csu, thank you for your question! 

I recommend opening an issue in [MLPrimitives](https://github.com/MLBazaar/MLPrimitives/issues) to request this feature.

**Comment by cjtaylo-csu** [comment]
Thank you @sarahmish , is this feature not currently supported?

I see a closed issue https://github.com/MLBazaar/MLPrimitives/issues/159 that appears similar.
But I don't see any guides referencing these callbacks. As I mentioned, my main goal is to plot the losses of the generators and discriminators of TadGAN during training.

Thank you

Chris

**Comment by cjtaylo-csu** [comment]
I now see that tadgan is calling its own .fit() function, not MLBlocks'.
https://github.com/sintel-dev/Orion/blob/master/orion/primitives/tadgan.py

I can clearly see the keras calls within. I can modify this function to plot losses during training.

Please let me know if you recommend any other method however.

Thanks

Chris
